.travis.yml

```diff
@@ -42,11 +41,19 @@ before_install:
     ${run_opts} ${distribution}-${version}:ansible "${init}"
 
 install:
-  - 'pip install -r requirements-devel.txt'
+  - 'sudo apt-get update -qq'
+  - 'sudo apt-get install -y python-pip'
+  - 'sudo pip install -r requirements-devel.txt'
+  - 'sudo pip install ansible'
```

Comment:
Cool. Travis does uses the virtualenv for each separate versions of Python. So no need to explicitly setup anything related to Python like pip.